symfonyTP/.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Les fichiers .idea n'ont rien a faire dans un dépot GIT, Il doivent être ignorer au niveau de la configuration de votre ordinateur : https://www.wikihow.com/Ignore-the-.Idea-Folder-from-Git-in-PhpStorm